torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5000,15 +5038,6 @@ def method_tests():
         ('tensor_split', (S, S, S), (3, 1), 'sections_dim', (False,), [1]),
         ('tensor_split', (S, S, S), ([2, 4],), 'indices', (False,)),
         ('tensor_split', (S, S, S), ([2, 4], 1), 'indices_dim', (False,), [1]),
-        ('scatter', (M, S), (0, gather_variable((S, S), 1, M), (S, S)), 'dim0', (), [0]),
-        ('scatter', (M, S), (1, gather_variable((M, S // 2), 0, S), (M, S // 2)), 'dim1', (), [0]),
-        ('scatter', (), (0, torch.tensor(0, dtype=torch.int64), ()), 'scalartensor_all_dim0', (), [0]),
-        ('scatter', (), (0, torch.tensor(0, dtype=torch.int64), 2.5), 'scalar_all_dim0', (), [0]),
-        ('scatter_add', (M, S), (0, gather_variable((S, S), 1, M), (S, S)), 'dim0', (), [0]),
-        ('scatter_add', (M, S), (1, gather_variable((M, S // 2), 0, S), (M, S // 2)), 'dim1', (), [0]),
-        ('scatter_add', (), (0, torch.tensor(0, dtype=torch.int64), ()), 'scalar_all_dim0', (), [0]),
```

Comment:
This case is missing in the new OpInfo tests

Reply:
I'm not sure I get which ones are missing. As far as I understand, all four of them are covered by `sample_inputs_scatter_ops`. Am I missing something?

- Line 5006: https://github.com/pytorch/pytorch/blob/dd64ee83ad6678aaeaa9368c64e2698eeef45628/torch/testing/_internal/common_methods_invocations.py#L2558-L2562
- Line 5007 and 5008: https://github.com/pytorch/pytorch/blob/dd64ee83ad6678aaeaa9368c64e2698eeef45628/torch/testing/_internal/common_methods_invocations.py#L2542-L2552
- Line 5009: https://github.com/pytorch/pytorch/blob/dd64ee83ad6678aaeaa9368c64e2698eeef45628/torch/testing/_internal/common_methods_invocations.py#L2554-L2556